src/App.js

```diff
@@ -14,11 +15,83 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    started: false,
+    goods: goodsFromServer,
```

Comment:
no need to keep goodsFromServer inside of our state, you can access them without storing them